### PR TITLE
[Display_01] Standardized name of SatelliteSignalCheckRequest

### LINKED ISSUE #143: [Display_01] Standardize SatelliteSignalCheckRequest Unit Tests Name
**Describe the bug**
Currently, the `DisplayName` of the `SatelliteSignalCheckRequestTests` reads `Sattelite Signal Check Request Tests` which doesn't match the other names and makes it more difficult to search the logs.

**Expected behavior**
For clarity of reading the unit test logs, can you please update the `DisplayName` of the `SatelliteSignalCheckRequestTests` class to `SatelliteSignalCheckRequest Unit Tests`? Thank you!

### PR DESCRIPTION
**Pull request by casher-ou**
Closes #143

### CODE REVIEW

File: src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import org.junit.jupiter.api.Test;
 
 @DisplayName("Sattelite Signal Check Request Tests")
-public class SatelliteSignalCheckRequestTests {
+public class SatelliteSignalCheckRequestUnitTests {
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java]: The class name was fine. That should be changed back.
```suggestion
public class SatelliteSignalCheckRequestTests {
```

File: src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import org.junit.jupiter.api.Test;
 
 @DisplayName("Sattelite Signal Check Request Tests")
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/SatelliteSignalCheckRequestUnitTests.java]: This was the part that needed to be updated.
```suggestion
@DisplayName("SatelliteSignalCheckRequest Unit Tests")
```

**Review by andrewdimmer — CHANGES_REQUESTED** [review]

**Review by andrewdimmer — APPROVED** [review]
Thank you, all set to merge!

**Review by stevecatherman — APPROVED** [review]
:)

**Pull request merged**